modules/imap/functions.php

```diff
@@ -17,16 +17,16 @@
  * @return array
  */
 if (!hm_exists('imap_sent_sources')) {
-function imap_sent_sources($callback, $mod) {
+function imap_sent_sources($callback, $mod, $folder = 'sent') {
```

Comment:
If the function is going to be used for any special directory, better use the name `imap_sources`